### PR TITLE
Remove project-json.md

### LINKED ISSUE #1513: Remove docs/core/tools/project-json.md

### PR DESCRIPTION
**Pull request by spboyer**
## Summary

remove project-json.md 

Fixes #1513 

## Suggested Reviewers
@mairaw @blackdwarf

### CODE REVIEW

**Review by thesmithee — APPROVED** [review]

**Review by mairaw — CHANGES_REQUESTED** [review]
we cannot remove topics. we should add the redirect metadata and redirect this topic somewhere. For an example, see https://github.com/dotnet/docs/blob/master/docs/core/app-types.md

**Comment by mairaw** [comment]
Also, remove that topic from the TOC please.

**Comment by spboyer** [comment]
@blackdwarf @mairaw - so redirect to where?

**Comment by mairaw** [comment]
@spboyer, @blackdwarf maybe here for now: https://docs.microsoft.com/en-us/dotnet/articles/core/preview3/tools/layering? Once we have the doc with Nate's blog post, then we could change to that one.

**Comment by spboyer** [comment]
@mairaw ok made the changes.  That appears to be the best place for now until @natemcmaster post is converted to a doc.

File: docs/core/tools/project-json.md [hunk 1]
```diff
@@ -1,1662 +1,8 @@
...
-For more information, see [Safe storage of app secrets during development](https://docs.microsoft.com/aspnet/core/security/app-secrets).
-
-For example:
-
-```json
-{
-    "userSecretsId": "aspnet-WebApp1-c23d27a4-eb88-4b18-9b77-2a93f3b15119"
-}
-```
+## Content moved!
```
Comment (mairaw) [inline docs/core/tools/project-json.md]: moved -> removed

File: docs/core/tools/project-json.md [hunk 2]
```diff
@@ -1,1662 +1,8 @@
...
+This content has merged and moved to the [High-level overview of changes in CLI RC3](../preview3/tools/layering.md) topic.
```
Comment (mairaw) [inline docs/core/tools/project-json.md]: This content is no longer relevant. For information on the changes, see the [High-level overview of changes in CLI RC4](../preview3/tools/layering.md) topic.

**Review by mairaw — CHANGES_REQUESTED** [review]
redirect looks good! some suggestions on the wording for people finding this on GH. Also, remove the entry from the TOC file.

**Comment by spboyer** [comment]
The following also link to this doc:

* docs/core/migrating-from-dnx.md
* docs/core/porting/libraries.md
* docs/core/tools/dotnet-build.md
* docs/core/tools/dotnet-pack.md
* docs/core/tools/dotnet-publish.md
* docs/core/tools/dotnet-restore.md
* docs/core/tools/dotnet-run.md
* docs/core/tools/extensibility.md

**Review by mairaw — APPROVED** [review]

**Comment by mairaw** [comment]
Thanks @spboyer! Yes, I'm aware of that. The only one we need to make sure is updated is the libraries one. The migrating-from-dnx should be removed by RTM and the tools folder content should be replaced by the tools content in the preview3 folder. In the meantime, those links will be redirected in the csproj branch.

**Comment by spboyer** [comment]
@mairaw is there an issue with Nate's blog > doc yet? And is there an ETA on it?  I have another item that I need to link to that on.

Thanks.

**Comment by mairaw** [comment]
No issues. Just other things that popped up @spboyer. But I've started the draft on Friday. I should have a PR soon.

**Comment by spboyer** [comment]
@mairaw ha, I meant a guthub issue (for tracking).

**Comment by mairaw** [comment]
Ha, completely misread that the first time. Yes, #1456 is the issue tracking that work.

**Pull request merged**